src/pages/Inventory/AddItem.js

```diff
@@ -84,19 +87,23 @@ const AddItem = () => {
                                     {" "}
                                     <FormattedMessage id="item.description"></FormattedMessage>
                                 </label>
-                                <textarea
+                                <Input
+                                    type="textarea"
                                     rows="4"
                                     name="description"
-                                    required="true"
+                                    required={true}
```

Comment:
no need to give a boolean value
```suggestion
                                    required
```